packages/build/src/plugins/resolve.js

```diff
@@ -155,6 +155,10 @@ const handleIntegrations = async function ({ integrations, autoPluginsDir, mode,
   const toInstall = integrations.filter((integration) => integration.has_build)
   await installIntegrationPlugins({ integrations: toInstall, autoPluginsDir, mode, logs })
 
+  if (toInstall.length === 0) {
```

Comment:
Nit, this one seems particularly excessive. If toInstall is an empty array we'll return Promise.all([]), which should be fine.

Reply:
Agreed actually, I'm doing some things here anyway on another branch so will tidy up a tad